Aneal-Mike-Project/babel/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    

```

Comment:
Your .html file could be improved by adding more semantic tags like `main` and `section` to give it a more proper structure to your code!

Consider checking this [article ](https://www.w3schools.com/html/html5_semantic_elements.asp)to achieve this! 😄

Reply:
- [ ]  [REQUIRED] You may use some introductory phrases hand like (e.g: I think it would be better if you do ....). This way you will avoid the Imperative Way point_right (e.g: giving Orders)